### ISSUE TITLE
flip dimensions in OrthoSlicer3D

### ISSUE THREAD

**Issue by pelednoam** [nipy/nibabel#559]
Hey,
I'm loading sample subject (from freesurfer) T1.mgz file, both using OrthoSlicer3D and freeview (same slice: 133, 84, 122). It looks like in the coronal and axial views, the left and right are flipped:

OrthoSlicer3D:

![orthoslicer3d](https://user-images.githubusercontent.com/1643819/30986294-ee7b08f4-a460-11e7-9533-3f3d42de48f1.png)

freeview:

![freeview](https://user-images.githubusercontent.com/1643819/30986328-0ce15910-a461-11e7-898e-3932c65fbe37.png)

Any idea why?
Thanks!

**Comment by effigies** [comment]
I'm replicating this issue, so I'd say it's either a bug in nibabel or in freeview...

**Comment by pelednoam** [comment]
I've loaded a CT of an epileptic patient with depth electrodes, and it can be seen clearly from the electrodes locations that it's a nibabel bug.

**Comment by effigies** [comment]
I'm going to generate a series of images with different orientations to check whether it's always LR flipped or just in some orientations. Hopefully it's always, because that'll be the easiest to fix. (The [logic](https://github.com/nipy/nibabel/blob/master/nibabel/viewers.py#L381-L409) of this feature is not easy to follow.)

**Comment by pelednoam** [comment]
Yes, I used this code to support slices view in my visualization software ([mmvt](http://mmvt.org)), and indeed, it was quite hard to follow. In the end, I've just flipped this dimension. From what I checked, it seems to be always LR flipped.

**Comment by effigies** [comment]
Thanks. Feel free to submit a PR with your modification. Otherwise I will, when I get a few more minutes.

**Comment by pelednoam** [comment]
Well, for now, in my code I've just added this line:
`flips[0] = not flips[0]
Not sure it's the best solution... Maybe a further investigation is needed to understand why the first item in flips is flipped on the first place...

**Comment by matthew-brett** [comment]
Sorry to be superficial - but can the difference be explained by differences in display conventions?  Nibabel does display the left on the left - neurological convention.  Maybe Freesurfer does right on the left - radiological?  Or am I missing something?

**Comment by matthew-brett** [comment]
@pelednoam - yes - the visualization code needs to be refactored.  It's been on my to-do list for a while ...

**Comment by effigies** [comment]
The L and R labels need to be swapped, then.

**Comment by pelednoam** [comment]
@matthew-brett, I don't think it's conventions issue, because both in nibabel and freeview, in those two perspectives, the label R is on the right and L is on the left. I could understand the flip if also the labels were flipped. 
Refactored or not, your code is really great, and saved me so much work! Thanks to you, I could add a slicer viewer into my Blender addon:

![image](https://user-images.githubusercontent.com/1643819/31581301-4ecf1eac-b136-11e7-9516-9c9a9ded6d84.png)

Thanks!
